nvtabular/column_group.py

```diff
@@ -41,29 +43,18 @@ def __init__(self, columns):
         self.kind = None
         self.dependencies = None
 
-        if isinstance(columns, str):
-            columns = [columns]
-
-        # if any of the values we're passed are a columngroup
-        # we have to ourselves as a childnode in the graph.
-        if any(isinstance(col, ColumnGroup) for col in columns):
-            self.columns = []
-            self.kind = "[...]"
-            for col in columns:
-                if not isinstance(col, ColumnGroup):
-                    col = ColumnGroup(col)
-                else:
-                    # we can't handle nesting arbitrarily deep here
-                    # only accept non-nested (str) columns here
-                    if any(not isinstance(c, str) for c in col.columns):
-                        raise ValueError("Can't handle more than 1 level of nested columns")
+        if isinstance(columns, list):
+            warnings.warn(
+                'The `["a", "b", "c"] >> ops.Operator` syntax for creating a `ColumnSelector` '
+                "has been deprecated in NVTabular 21.09 and will be removed in a future version.",
+                FutureWarning,
+            )
+            columns = ColumnSelector(columns)
```

Comment:
Isn't this supposed to say that we are deprecating `ColumnGroup`, rather than `ColumnSelector`?

If so, we should probably add the same text to the `ColumnGroup` docstring (saying something along the lines of: "please use `ColumnSelector(...)`

Reply:
Good catch on the text! I don't think we want to deprecate the `ColumnGroup` class (which will become `WorkflowNode` in the next set of changes), just this particular syntax.